tianshou/env/vecenv/shmemenv.py

```diff
@@ -60,23 +58,23 @@ def _encode_obs(maybe_dict):
 
 
 class ShmemVectorEnv(SubprocVectorEnv):
-    """Optimized version of SubprocVectorEnv that uses shared
-    variables to communicate observations.
+    """Optimized version of SubprocVectorEnv that uses shared variables to
+    communicate observations.
 
     .. seealso::
 
         Please refer to :class:`~tianshou.env.BaseVectorEnv` for more detailed
         explanation.
 
     I borrowed heavily from openai baseline to implement ShmemVectorEnv Class.
-    Please refer to 'https://github.com/openai/baselines/blob/master/baselines/
-    common/vec_env/shmem_vec_env.py' for more info if you are interested.
+    Please refer to "https://github.com/openai/baselines/blob/master/baselines/
+    common/vec_env/shmem_vec_env.py" for more info if you are interested.
+
                     Huayu Chen(chenhuay17@mails.tsinghua.edu.cn)  2020/08/01
     """
 
     def __init__(self, env_fns: List[Callable[[], gym.Env]]) -> None:
         BaseVectorEnv.__init__(self, env_fns)
-        self.closed = False
```

Comment:
I don't think this should be deleted. Since SubprocVectorEnv is not initialized.

Reply:
Oh my bad